{"name": "test06", "status": "passed", "description": "assert that you get a exception when you type invalid email adress", "start": 1710320519133, "stop": 1710320521723, "uuid": "cd295b76-a65f-4514-80b0-cc94b41e08bc", "historyId": "b9b530c5cc25f8123ba9c34a7ef66853", "testCaseId": "b9b530c5cc25f8123ba9c34a7ef66853", "fullName": "test_cases.Test_cases#test_assert_exception_message_when_type_invalid_email", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('init_webdriver')"}, {"name": "suite", "value": "test_cases"}, {"name": "subSuite", "value": "Test_cases"}, {"name": "host", "value": "LAPTOP-C1F79DIL"}, {"name": "thread", "value": "6048-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_cases"}]}